tests/security/selinux/selinux_setup.pm

```diff
@@ -59,16 +64,18 @@ sub run {
             'selinux-policy-minimum-20220124-150400.121.1.noarch.rpm',
             'selinux-policy-devel-20220124-150400.121.1.noarch.rpm'
         );
```

Comment:
@ShawnHa0 
Given the fact that you have already put all pkgs in a tarball.
I suggest you remove these lines and later
``` #rpm -ivhU --nosignature --nodeps --noplugins ./*.rpm